packages/nimble-components/src/anchor-tabs/tests/anchor-tabs.spec.ts

```diff
@@ -203,6 +204,20 @@ describe('AnchorTabs', () => {
             initialFocusIndex: 0,
             keyName: keyEnd,
             expectedFinalFocusIndex: 1
+        },
+        {
+            name: 'should skip hidden tab when arrowing right',
```

Comment:
The test cases for hidden tabs don't seem quite as complete as the test cases for disabled tabs. Is there a reason not to have a test for arrowing left or for pressing the Home key?

Reply:
I know they exercise the same relevant code paths, so I didn't write them. But then the same logic would apply to disabled tabs... so I guess I should add more test cases for consistency.